src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -1,17 +1,24 @@
 package org.sonar.plugins.stash;
 
 import java.util.Optional;
+import java.io.File;
+import java.text.MessageFormat;
+import java.util.ArrayList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)